src/server/adminInviteUser.ts

```diff
@@ -48,30 +49,82 @@ export async function adminInviteUser(
         redirectTo: redirectUrl,
     });
 
+    const auditLog = {
+        action: "send invite link to user by email",
+        content: {
+            email: userDetails.email,
+        },
+    } as const satisfies Partial<AuditLog>;
+
     if (error) {
-        const logId = await logErrorReturnLogId("Error with inviting user", { error: error });
+        const logId = await logErrorReturnLogId("failed to invite user", { error: error });
+        await sendAuditLog({
+            ...auditLog,
+            wasSuccess: false,
+            logId,
+        });
         return {
             data: null,
             error: { type: "inviteUserFailure", logId: logId },
         };
     }
 
-    const { error: createRoleError } = await supabase.from("profiles").insert({
-        role: userDetails.role,
-        first_name: userDetails.firstName,
-        last_name: userDetails.lastName,
-        telephone_number: userDetails.telephoneNumber,
-        user_id: data.user.id,
+    await sendAuditLog({
+        ...auditLog,
+        wasSuccess: true,
+        content: {
+            ...auditLog.content,
+            newUserId: data.user.id,
+            invitedAt: data.user.invited_at,
+            createdAt: data.user.created_at,
+        },
     });
 
-    if (createRoleError) {
-        const logId = await logErrorReturnLogId("Error with insert profile", {
-            error: createRoleError,
+    if (data) {
```

Comment:
can we rename `data` here so we know what data that is? since this function is semi-long and does a few things